stack.yaml

```diff
@@ -39,10 +39,17 @@ packages:
 - '.'
 # Dependency packages to be pulled from upstream that are not in the resolver
 # (e.g., acme-missiles-0.3)
-extra-deps: []
-
+extra-deps: 
```

Comment:
If possible, it might make more sense to update this to amazonka 1.6.1 deps from hackage, than depending on github revision.